rad/prelude/bool.rad

```diff
@@ -1,3 +1,9 @@
+'prelude/bool
+
+"Functions for dealing with truthiness and #f."
```

Comment:
Not sure if at the RFC stage I should be talking only about broader points than syntax, but 

```
(module 'prelude/bool '[not and or all]
  "Functions for ...") 
```
Looks prettier and less magical to me.

Reply:
Yeah I agree, but I used `module` as the special-form to create modules inline. Also I want to keep files executable even if they aren't being treated as modules. So e.g.:

```clojure
`[:module prelude/bool [not and or all]
  "Functions for ..."]
```
?

Reply:
I guess we could just define a no-op module function, but I actually like the vector better.

Reply:
Hm, in the interest of making it easier to have backwards compatibility, maybe a dict? E.g.:
```
{:module 'prelude/bool
 :doc "Functions for ..."
 :exports '[not and or all]
}